io/disk/tiobench.py.data/tiobench.yaml

```diff
@@ -0,0 +1,26 @@
+setup: 
+    location: !mux
+        default:
+            target: '/mnt'
+
+    block: !mux
+        first:
+            blocks: 4096
+        second:
+            blocks: 8192
+
+    thread: !mux
+        default:
+            threads: 10
+
+    blocksize: !mux
+        default: 
```

Comment:
there are lots of empty characters in this file, please remove them. (for example at the end of this line)